code/__DEFINES/hud.dm

```diff
@@ -34,10 +34,19 @@
 	Therefore, the top right corner (except during admin shenanigans) is at "15,15"
 */
 
-/proc/ui_hand_position(i) //values based on old hand ui positions (CENTER:-/+16,SOUTH:5)
-	var/x_off = i % 2 ? 0 : -1
-	var/y_off = round((i-1) / 2)
-	return"CENTER+[x_off]:16,SOUTH+[y_off]:5"
+/**
+ * values based on old hand ui positions (CENTER:-/+16,SOUTH:5)
+ * If item arg is set, its base_pixel_x and base_pixel_y will be added to the pixel offsets of the return value.
+ */
+/proc/ui_hand_position(index, obj/item/item)
+	var/x_off = -(!(index % 2))
```

Comment:
the fuck is this

Reply:
No fucking idea. It's one of those things that have already been there that I dared not to touch.

Reply:
the old code had an improved version, you should use it instead.
I ran into it when I was touching action buttons and moth made me fix it

Reply:
Oh. Looks like I accidentally regressed the code while resolving the conflicts from the action buttons PR. Yes, I'll fix that.